content/en/deployment/hpa.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Horizontal Pod Autoscaling in CD-as-a-Service
+linktitle: HPA
+weight: 10
+description: >
+  How CD-as-a-Service handles Horizontal Pod Autoscaling during deployments.
```

Comment:
```suggestion
 Learn how CD-as-a-Service handles Kubernetes Horizontal Pod Autoscaling during deployments.
```

Adding keyword "Kubernetes" to description. 
Note: I'll review all `title` and `desc` with Anna/Alicia to ensure we hit our SEO marks